packages/lib-sourcify/src/lib/solidityCompiler.ts

```diff
@@ -109,12 +137,14 @@ function validateSolcPath(solcPath: string): boolean {
   return false;
 }
 
```

Comment:
```suggestion

/**
 * Fetches a solc binary from GitHub and saves it to the given path.
 * 
 * If platform is "bin", it will download the solc-js binary.
 */
```